### ISSUE TITLE
Review validation file

### ISSUE THREAD

**Issue by hungdodang** [namtranase/generative_model_biosignal#7]
- [x]  Check calculate hr and rr function and finding why hr = rr 

- [x]  Train model with data after noise filter (T sensie's order)

- [ ]  When rr calculating func seems right, test with trained model. Also try validating with RMS, PRD

- [ ]  Answer some questions of T sensei: how the updating process with Gradient work; how to choose number of LSTM cell, kernel size,... (or base on what); with the given formula for loss why using cross entropy.

**Comment by hungdodang** [comment]
![genereated_signal](https://user-images.githubusercontent.com/63043543/101276512-f904ad00-37df-11eb-876d-2e8cc2e05f6b.PNG)
Generated signal of model after 200 epochs

**Comment by hungdodang** [comment]
![image](https://user-images.githubusercontent.com/63043543/101499729-2ef08f80-39a0-11eb-9b66-c173550292d7.png)
Generated signal and original signal do not have same size => maybe it is the reason why RR = HR for generated signals!
Function for hr and rr work well with original signal read from csv file

**Comment by hungdodang** [comment]
[https://datascience.stackexchange.com/questions/47875/why-is-my-generator-loss-function-increasing-with-iterations](url)
Reason why G loss go up!

**Comment by namtranase** [comment]
> [https://datascience.stackexchange.com/questions/47875/why-is-my-generator-loss-function-increasing-with-iterations](url)
> Reason why G loss go up!

Nice job!
